docs/guides/troubleshooting/advanced-troubleshooting-dns-problems/index.md

```diff
@@ -0,0 +1,250 @@
+
+### Windows
+
+1.  Right-click the network icon in the **Notification Area** of the **Taskbar**.
+
+2.  Choose **Network and Internet Settings**.
+
+3.  Select your network adapter.
+
+4.  Scroll down to reveal the settings for that network adapter.
+
+### macOS
+
+1.  Click the network icon in the **Menu Bar**.
+
+2.  Click **Network Preferences**.
+
+3.  Select your network adapter.
+
+4.  Click **Advanced**.
+
+5.  Click **DNS** to reveal the settings for that adapter.
+
+### Linux
+
+In Linux, the location of the desired DNS server depends on the values set in the resolver configuration file located at `/etc/resolve.conf`.
+
+To find these values via the command line, enter:
+
+    cat /etc/resolve.conf
+
+To find these values via the GUI in Ubuntu 22.04 LTS:
+
+1.  Click network icon in the GNOME Panel.
+
+2.  Select your network adapter.
+
+3.  Click **<adapter/connection/device> Settings**.
+
+4.  Click the gear icon next to your chosen adapter/connection/device
+
+Knowing the desired address helps trace the circuit to determine if the desired DNS server can be reached.
+
+## Circuit Testing with Ping
+
+Troubleshooting the network communications circuit uses common tools like `ping`. Many queries fail because there is no network circuit to the desired DNS server. The `ping` command line tool is universal to all operating systems and does not require administrative rights.
+
+Open a command line and use `ping` to determine if the IP address responds. In this case, the public DNS server for Cloudflare (`1.1.1.1`) is cited:
+
+    ping 1.1.1.1
+
+Or:
+
+    ping cloudflare.com
+
+When a reply is returned, queries can be made to the server, in which case, skip to the next section.
+
+If there is no reply using `ping`, then the circuit is unusable and the requested resolver is not responding. This can be because it’s under attack, dead, or offline. Specify another resolver/DNS server and repeat the attempt.
+
+An unusable circuit, or route to a DNS server, means that an Internet connection is down for the host making queries. If the circuit is up, another DNS server can be used, and must be placed in the host's settings. Reset the host to restart the network stack and read the new DNS server entry. Such alternate DNS Internet servers are `1.1.1.1` for Cloudflare, and `8.8.8.8` for Google.
+
+{{< note >}}
+The `traceroute` command line tool is also handy in network circuit analysis. It uses either an IP address or a Fully Qualified Domain Name (FQDN) to show latencies between the host and the desired IP address/FQDN.
+{{< /note >}}
+
+## Tracing Resolvers with NSLookup
+
+Windows, macOS, and Linux contain the command line tool, `nslookup` (Name Server Lookup). It performs a query on a known address, using the default DNS resolver, unless another resolver is specified.
+
+    nslookup google.com
+
+Renders:
+
+{{< output >}}
+    C:\Users\henry10>nslookup google.com
```

Comment:
We can remove instances of the command prompt in output, like this `C:\Users\henry10>`.